dataprofiler/profilers/profiler_options.py

```diff
@@ -886,13 +890,28 @@ def __init__(
         :ivar stop_condition_unique_value_ratio: The highest ratio of unique
             values to dataset size that is to be considered a categorical type
         :vartype stop_condition_unique_value_ratio: [None, float]
+        :ivar cms: boolean option for using count min sketch
+        :vartype cms: bool
+        :ivar cms_confidence: defines the number of hashes used in CMS.
+            eg. confidence = 1 - failure probability, default 0.95
+        :vartype cms_confidence: [None, float]
+        :ivar cms_relative_error: defines the number of buckets used in CMS,
+            default 0.01
+        :vartype cms_relative_error: [None, float]
+        :ivar heavy_hitters_threshold: minimum proportion of total count for CMS
+            to store a distinct category.
+        :vartype heavy_hitters_threshold: [None, int]
         """
         BaseInspectorOptions.__init__(self, is_enabled=is_enabled)
         self.top_k_categories = top_k_categories
         self.max_sample_size_to_check_stop_condition = (
             max_sample_size_to_check_stop_condition
         )
         self.stop_condition_unique_value_ratio = stop_condition_unique_value_ratio
+        self.cms = cms
```

Comment:
as opposed to cms = True or False.

I think it might be better to use a `method` param
which chooses `"full" | "cms".

open to thoughts or opposition here.

Reply:
It's currently just being used as an on/off switch. Useful to have param method if there's many options to choose from. Are there plans to expand?

Reply:
Was just trying to match the same technique we used for `unique_count` which makes it extensible and more clear wrt what the other option you are choosing or have set as an option when reading code or options.